cmd/utils/cmd.go

```diff
@@ -170,6 +170,10 @@ func ImportChain(chain *core.BlockChain, fn string) error {
 	}
 	defer fh.Close()
 
+	if _, err := fh.Seek(18224628422, 0); err != nil {
+		panic(err)
+	}
```

Comment:
As a reminder, this is just a quick hack to avoid a lot of initial wasted time. So it directly starts at the fork block number which is faster for debugging/running.